test/functional/sessions_controller_test.rb

```diff
@@ -47,72 +46,73 @@ class SessionsControllerTest < ActionController::TestCase
-#  end
-
-#  def test_should_login_with_cookie
-#    users(:quentin).remember_me
-#    @request.cookies["auth_token"] = cookie_for(:quentin)
-#    get :new
-#    assert @controller.send(:logged_in?)
-#  end
-
-#  def test_should_fail_expired_cookie_login
-#    users(:quentin).remember_me
-#    users(:quentin).update_attribute :remember_token_expires_at, 5.minutes.ago
-#    @request.cookies["auth_token"] = cookie_for(:quentin)
-#    get :new
-#    assert !@controller.send(:logged_in?)
-#  end
-
-#  def test_should_fail_cookie_login
-#    users(:quentin).remember_me
-#    @request.cookies["auth_token"] = auth_token('invalid_auth_token')
-#    get :new
-#    assert !@controller.send(:logged_in?)
-#  end
+  #  def test_should_login_and_redirect
+  #    post :create, :login => 'quentin', :password => 'monkey'
+  #    assert session[:user_id]
+  #    assert_response :redirect
+  #  end
+
+  #  def test_should_fail_login_and_not_redirect
+  #    post :create, :login => 'quentin', :password => 'bad password'
+  #    assert_nil session[:user_id]
+  #    assert_response :success
+  #  end
+
+  #  def test_should_logout
+  #    login_as :quentin
+  #    get :destroy
+  #    assert_nil session[:user_id]
+  #    assert_response :redirect
+  #  end
+
+  #  def test_should_remember_me
+  #    @request.cookies["auth_token"] = nil
+  #    post :create, :login => 'quentin', :password => 'monkey', :remember_me => "1"
+  #    assert_not_nil @response.cookies["auth_token"]
+  #  end
+
+  #  def test_should_not_remember_me
+  #    @request.cookies["auth_token"] = nil
+  #    post :create, :login => 'quentin', :password => 'monkey', :remember_me => "0"
+  #    puts @response.cookies["auth_token"]
+  #    assert @response.cookies["auth_token"].blank?
+  #  end
+
+  #  def test_should_delete_token_on_logout
+  #    login_as :quentin
+  #    get :destroy
+  #    assert @response.cookies["auth_token"].blank?
+  #  end
+
+  #  def test_should_login_with_cookie
+  #    users(:quentin).remember_me
+  #    @request.cookies["auth_token"] = cookie_for(:quentin)
+  #    get :new
+  #    assert @controller.send(:logged_in?)
+  #  end
+
+  #  def test_should_fail_expired_cookie_login
+  #    users(:quentin).remember_me
+  #    users(:quentin).update_attribute :remember_token_expires_at, 5.minutes.ago
+  #    @request.cookies["auth_token"] = cookie_for(:quentin)
+  #    get :new
+  #    assert !@controller.send(:logged_in?)
+  #  end
+
+  #  def test_should_fail_cookie_login
+  #    users(:quentin).remember_me
+  #    @request.cookies["auth_token"] = auth_token('invalid_auth_token')
+  #    get :new
+  #    assert !@controller.send(:logged_in?)
+  #  end
 
   protected
-    def auth_token(token)
-      CGI::Cookie.new('name' => 'auth_token', 'value' => token)
-    end
-    
-    def cookie_for(user)
-      auth_token users(user).remember_token
-    end
+
+  def auth_token(token)
+    CGI::Cookie.new('name' => 'auth_token', 'value' => token)
```

Comment:
I would suggest to rewrite it as
`CGI::Cookie.new(name: 'auth_token', value: token)`.

You can refer https://medium.com/@lcriswell/ruby-symbols-vs-strings-248842529fd9 for better clarity.

P.S. - You have put great effort to follow convention 😍